src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -73,7 +73,7 @@ export class CalciteActionBar {
   /**
    * Arranges the component depending on the elements 'dir' property.
    */
-  @Prop({ reflect: true }) position: CalcitePosition;
+  @Prop({ reflect: true }) position: CalcitePosition = "start";
```

Comment:
Do we need this to have a value or can we assume `start` if not present?

Reply:
I am using that attribute in the styles in center-row.

Reply:
Although...yeah. I see. I'll update.

Reply:
@driskull Done. Thanks.